test/web3c.js

```diff
@@ -4,6 +4,7 @@ const web3 = require('web3');
 const web3c = require('../');
 const gateway = require('./mockgateway');
 const artifact = require('../demo/example.json');
+const responses = require('./mockgateway/responses');
```

Comment:
you both include responses directly, and use the re-exported gateway.responses in this file. choose one.

Reply:
Removed.